server/dist/container.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
```

Comment:
All the /dist files shouldn't be pushed to the git repo. Just add them to `.gitignore`.